webrtc-stats.html

```diff
@@ -2096,6 +2117,36 @@
     </section>
     <section>
       <h2>
+        Obsolete stats
+      </h2>
+      <p>
+        This section's content is an example.
```

Comment:
We have a concrete example, rtt got deprecated for roundTripTime, handle that as a separate issue?